src/app/modules/form/form.utils.ts

```diff
@@ -65,3 +66,13 @@ export const getCollabEmailsWithPermission = (
     return acc
   }, [])
 }
+
+/**
+ * Type guard for whether a populated form is email mode
+ * @param form Form document to check
+ */
+export const isEmailModeForm = (
```

Comment:
there's actually already an `isEmailModeForm` function in `src/types/form`

Reply:
As discussed, this was moved from types to utils, since it is a function and not a type.